src/parser/jobs/ast/BuffExtensions.module.css

```diff
@@ -0,0 +1,24 @@
+.buffextensions__headerItem {
```

Comment:
BEM :blobknife:

No need to namespace css classes, the names get mangled to include module info and a hash during build.